modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/resources/META-INF/resources/user_associated_data_entity_action.jsp

```diff
@@ -31,7 +31,8 @@ UADEntity uadEntity = (UADEntity)row.getObject();
 
 	<liferay-ui:icon
 		message="auto-anonymize"
-		url="<%= autoAnonymizeURL %>"
+		onClick='<%= renderResponse.getNamespace() + "confirmAction('" + autoAnonymizeURL.toString() + "', '" + UnicodeLanguageUtil.get(request, "are-you-sure-you-want-to-anonymize-this-entity") + "')" %>'
+		url="javascript:;"
```

Comment:
I don't think this icon is actually inside a form.  This page is just rendered inside a search container, right?  Does the function still work here?

Reply:
Yeah I just checked, you get a JS TypeError because the `form` variable is null when the JS function tries to call `form.attr('method', 'post');`.

Reply:
a whups - I checked if the box shows up, but not if it completed. I can fix that and push up to this branch if you want. As far as the UnicodeLanguageUtil - it's just the one that I saw being used